android/app/src/main/java/com/reactnativenavigation/activities/BottomTabActivity.java

```diff
@@ -111,12 +111,12 @@ public void push(Screen screen) {
     public Screen pop(String navigatorId) {
         super.pop(navigatorId);
         Screen screen = mScreenStacks.get(mCurrentStackPosition).pop();
-        setNavigationStyle(screen);
+        setNavigationStyle(getCurrentScreen());
```

Comment:
This was being called with the screen that was being popped rather than the screen being changed to. Not sure if that was an existing issue or if I've misinterpreted what was going on.